src/App.js

```diff
@@ -1,24 +1,19 @@
 import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
```

Comment:
let's keep App.css :)

Reply:
@udenizdemirbilek Thank you for feedback. I'll close PR. Because I think this is not last version that I pull requested. I'll do it again. Also I'll solve what you've said!